services/identity/service.go

```diff
@@ -373,6 +388,17 @@ func newIdentityService(c *sda.Context, path string) sda.Service {
 		skipchain:        skipchain.NewClient(),
 		path:             path,
 	}
+	var err error
+	s.propagateIdentity, err =
+		manage.NewPropagationFunc(c, "IdentityPropagateID", s.propagateIdentityHandler)
+	s.propagateSkipBlock, err =
+		manage.NewPropagationFunc(c, "IdentityPropagateSB", s.propagateSkipBlockHandler)
+	s.propagateConfig, err =
+		manage.NewPropagationFunc(c, "IdentityPropagateConf", s.propagateConfigHandler)
+	if err != nil {
```

Comment:
Theses errors only get caught if the last call of `manage.NewPropagationFunc` returns an error. Otherwise the `err` variable gets overwritten by subsequent calls above.